package.json

```diff
@@ -17,6 +17,7 @@
   "dependencies": {
     "@gnosis.pm/dex-contracts": "^0.4.1",
     "@gnosis.pm/dex-liquidity-provision": "gnosis/dex-liquidity-provision#cmm-changes",
+    "@gnosis.pm/dex-js": "^0.4.1",
```

Comment:
hehe, nice u make use of it!

Reply:
Oh yes, and I intend at some point to add more stuff to it.
Would you be willing to have more contracts added to `dex-js`? :nerd_face:

Reply:
It doesn't bother me if they are GP related why not :) 
we don't need to import them in the projects, so as long as they are tree-shakeable is good